Sources/Views/MessagesCollectionView.swift

```diff
@@ -41,6 +41,10 @@ open class MessagesCollectionView: UICollectionView {
         return messagesCollectionViewFlowLayout.isTypingIndicatorViewHidden
     }
 
+    /// Display the date of message by swiping left.
+    /// The default value of this property is `false`.
+    open var displayTimeBySwipingLeft: Bool = false
```

Comment:
It seems like `MessagesViewController.displayTimeBySwipingLeft` is the way to enable this feature. Should this property be for internal use only then? It may confuse clients what property to set to get this feature working